web/js/tour.js

```diff
@@ -236,7 +236,7 @@ export default function (models, ui, config) {
       .width();
     var viewHeight = $(window)
       .height();
-    return viewWidth >= 768 && viewHeight >= 680;
+    return viewWidth > 800 && viewHeight > 680;
```

Comment:
Is this for iPad specifically?

Reply:
@minniewong was seeing this on iPad 1st Gen portrait view 😮, I checked the resolution of this device and made the change to greater than 800 to prevent the dialog from occurring on portrait modes where the width is more 768px (iPad) or 800px on some other devices.